docs/source/networking/kaizen-networks.md

```diff
@@ -9,64 +9,38 @@ IP Address       Hostname/Description
 129.10.5.1       Router IP (on cisco switch) (Highly Available)
 129.10.5.2       Router IP (on cisco switch)
 129.10.5.3       Router IP (on cisco switch)
-129.10.5.4       kzn-hil-client.infra.massopen.cloud
 
-129.10.5.40      dns server for openshift subdomains (openapp.cloud, mocapps.cloud, massopen.cloud test subdomains)
-129.10.5.41      dns server for massopen.cloud
-
-129.10.5.55      dhcp1.cnv.massopen.cloud
-129.10.5.56      dhcp2.cnv.massopen.cloud
+129.10.5.41      ns2.massopen.cloud (dns server for massopen.cloud)
 
 129.10.5.101     ov1.massopen.cloud
 129.10.5.102     ov2.massopen.cloud
 129.10.5.103     ov3.massopen.cloud 192.12.185.247 *129.10.5.103* is local only
 129.10.5.104     ovirt.massopen.cloud (Web interface)
 129.10.5.105     kzn-ipmi-gw.infra.massopen.cloud
 129.10.5.106     kzn-rtr.infra.massopen.cloud (for all subnets)
-129.10.5.107     kzn-ssh.infra.massopen.cloud
-129.10.5.108     mochat.infra.massopen.cloud (shared repo server)
-129.10.5.109     kzn-nagios.infra.massopen.cloud
-129.10.5.110     rhel-6-repomirror.massopen.cloud
 
-129.10.5.114     wl.massopen.cloud (to be deleted)
 129.10.5.115     tld.massopen.cloud
 129.10.5.116     freeipa.infra.massopen.cloud
-129.10.5.117     netbox.massopen.cloud
+
 129.10.5.118     techsquare.massopen.cloud
 129.10.5.119     dns.massopen.cloud
-129.10.5.120     helpdeskvm.massopen.cloud (to be deleted)
-129.10.5.121     massopen.cloud
-129.10.5.123     reports.infra.massopen.cloud (to be deleted)
-129.10.5.124     kzn-ipmi2-gw.infra.massopen.cloud
-
-129.10.5.127     kzn-hil.massopen.cloud (to be deleted)
-129.10.5.128     kzn-ovpn.massopen.cloud
 
 129.10.5.132     zabbix.massopen.cloud
```

Comment:
Didn't we get rid of zabbix?

Reply:
we haven't yet. It's still monitoring the ovirt cluster, ceph hosts and other random things.